lib/client.js

```diff
@@ -792,62 +793,46 @@ class Client extends events.EventEmitter {
     }
 
     /**
-     * Connects and handles the execution of prepared and simple statements.
+     * Wrapper for executing queries by rust driver
```

Comment:
Please check the rest of the private methods. They do not have arguments listed in JSDoc.